pkgs/by-name/di/diesel-cli-ext/package.nix

```diff
@@ -4,7 +4,8 @@
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "diesel-cli-ext";
+  name = "diesel-cli-ext";
```

Comment:
Instead of setting `name`, set `meta.mainProgram` to fix the `nix run` issue.

Reply:
Thanks!